### ISSUE TITLE
Apple LLVM 9.0 Error (Firestore)

### ISSUE THREAD

**Issue by rawandahmad698** [firebase/firebase-ios-sdk#1702]
Regarding issue [#1266](https://github.com/firebase/firebase-ios-sdk/issues/1266)

I went ahead and commented the` #import "transformations/GRXMappingWriter.h"` line from `gRPC_RxLibrary-umbrella.h` as @paulb777 workaround. But getting this. 

[Screenshot_1](http://prntscr.com/kj9q63)

### [REQUIRED] Step 2: Describe your environment

  * Xcode version: 9.2
  * Firebase SDK version: Latest
  * Firebase Component: (Auth, Core, Database, Firestore)
  * Component version: 5.6.0

### [REQUIRED] Step 3: Describe the problem
I want to upload my app to the AppStore, but i can't archive the project
#### Steps to reproduce:

PodFile: 

```
pod 'Firebase/Core'
pod 'Firebase/Database'
pod 'Firebase/Messaging'
pod 'Firebase/Firestore'
pod 'Firebase/Auth'
pod 'GRDB.swift'
pod 'Fabric', '~> 1.7.9'
pod 'Crashlytics', '~> 3.10.5'
pod 'gRPC', :git => 'https://github.com/grpc/grpc.git', :branch => 'master'
pod 'gRPC-Core', :git => 'https://github.com/grpc/grpc.git', :branch => 'master'
pod 'gRPC-ProtoRPC', :git => 'https://github.com/grpc/grpc.git', :branch => 'master'
pod 'gRPC-RxLibrary', :git => 'https://github.com/grpc/grpc.git', :branch => 'master'
```

What happened? How can we make the problem occur?

#### Relevant Code:

Make a new project, Use this podfile and archive the app

```
// TODO(you): code here to reproduce the problem
```

**Comment by paulb777** [comment]
The workarounds should no longer be needed since the issue was fixed in gRPC 1.13.0.

We recommend strongly against ever using `:branch => 'master'` for anything except evaluation, since the code there may be work in progress and not fully integration tested.

With the latest release, there should be no need to specify any of the gRPC pods in the Podfile since the Firestore dependencies will bring in the right ones.

**Comment by rawandahmad698** [comment]
@paulb777. Thanks, fixed
